src/setup/txt_joybinput.c

```diff
@@ -98,6 +99,9 @@ static void CanonicalizeButtons(void)
     int vbutton;
     int i;
 
+    static_assert(arrlen(all_joystick_buttons) == NUM_VIRTUAL_BUTTONS,
```

Comment:
Isn't this a long way around of just doing this?:
`static int *all_joystick_buttons[NUM_VIRTUAL_BUTTONS] = ...`